### PR TITLE
#1853 - Create the route GET /ga4/open-email-tracking.

### PR DESCRIPTION
**Pull request by kalbfled**
# Description

These changes add the route GET /ga4/open-email-tracking, which expects URL parameters.  The code builds a dictionary from those parameters and validates them against a JSON schema.  This route does not use authentication.

issue #1853

## Type of change

Please check the relevant option(s).

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation changes only

## How Has This Been Tested?

- [x] Deploy to Dev, make a command line request to the public end point, and inspect the logs

image

## Checklist

- [x] I have assigned myself to this PR
- [x] PR has an [appropriate title](https://github.com/department-of-veterans-affairs/vanotify-team/blob/master/Engineering/team_agreement.md#naming-prs): #9999 - What the thing does
- [x] PR has a detailed description, including links to specific documentation
- [x] I have added the [appropriate labels](https://github.com/department-of-veterans-affairs/notification-api/blob/master/.github/release.yaml) to the PR.
- [x] I did not remove any parts of the template, such as checkboxes even if they are not used
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to any documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have ensured the latest master is merged into my branch and all checks are green prior to review
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] The ticket is now moved into the DEV test column

### CODE REVIEW

**Comment by cris-oddball** [comment]
Also, please update the label to Enhancement.

File: app/googleanalytics/ga4.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+ga4_blueprint = Blueprint('ga4', __name__, url_prefix='/ga4')
+
+ga4_request_validator = Draft202012Validator(ga4_request_schema, format_checker=FormatChecker(['uuid']))
+
+
+@ga4_blueprint.route('/open-email-tracking', methods=['GET'])
+def get_ga4():
+    """
+    This route is what alerts to the opening of an e-mail message.
```
[the remark this answers is not in the document]
Comment (kalbfled) [reply app/googleanalytics/ga4.py]: Updated to something similar to what you wrote

[the remark this answers is not in the document]
Comment (cris-oddball) [reply app/googleanalytics/ga4.py]: Much better wording than mine! ty!

**Review by cris-oddball — APPROVED** [review]

**Review by MackHalliday — APPROVED** [review]
Reviewed looks good!

**Review by EvanParish — APPROVED** [review]

**Pull request merged**